contrib/internal/sqltest/sqltest.go

```diff
@@ -88,9 +108,9 @@ func testQuery(cfg *Config) func(*testing.T) {
 		assert.Nil(err)
 
 		spans := cfg.mockTracer.FinishedSpans()
-		assert.Len(spans, 1)
+		assert.Len(spans, 2)
```

Comment:
Same question here: do we not want to make sure all of the spans are what we expect? Checking just one is maybe not enough for a thorough test coverage.